lib/ansible/modules/net_tools/exoscale/exo_rdns_record.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2019, Lorenz Schori <lo@znerol.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: exo_rdns_record
+short_description: Manages reverse DNS records for Exoscale compute instances.
+description:
+    - Set and unset reverse DNS record on Exoscale instance.
+version_added: "2.9"
+author: "Lorenz Schori (@znerol)"
+options:
+  name:
+    description:
+      - Name of cs instance.
+    required: true
+    type: str
+  content:
+    aliases: [ 'value' ]
+    description:
+      - Reverse DSN name for the cs instance. Required if state=present.
+    type: str
+  state:
+    description:
+      - State of the record.
+    default: present
+    choices: [ 'present', 'absent' ]
+    type: str
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+- name: Set the reverse DNS for a virtual machine
+  local_action:
+    name: web-vm-1
+    content: www.example.com
+
+- name: Delete the reverse DNS for a virtual machine
+  local_action:
+    name: web-vm-1
+    state: absent
+'''
+
+RETURN = '''
+---
+exo_rdns_domain:
+  description: Reverse DSN name for the cs instance
+  returned: success
+  type: str
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_bytes, to_text
+from ansible.module_utils.cloudstack import (
+    AnsibleCloudStack,
+    cs_argument_spec,
+    cs_required_together
+)
+
+
+class ExoRdnsRecord(AnsibleCloudStack):
+
+    def __init__(self, module):
+        super(ExoRdnsRecord, self).__init__(module)
+
+        self.name = self.module.params.get('name')
+        self.content = self.module.params.get('content')
+        if self.content[-1] != ".":
+            self.content += "."
+
+        self.returns = {
```

Comment:
IMHO this is unused, right?

Reply:
Right, according to the [docs](https://github.com/ansible/ansible/pull/57319).